.github/workflows/check-unity-sdk-for-updates.yml

```diff
@@ -0,0 +1,55 @@
+name: Check Unity SDK Updates
+
+on:
+  schedule: # Run Hourly
+    - cron: '0 * * * *'
```

Comment:
just curious if SDK are released that often maybe is better to change this daily or weekly

Reply:
I don't think they're released that often, but I don't like the idea of having this package out of sync with the available updates for so long.